
2023-05-21 17:40:28 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:40:28 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:52:50 ERROR: 127.0.0.1 - Exception on /url/urls [GET]
Traceback (most recent call last):
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_restx/api.py", line 404, in wrapper
    resp = resource(*args, **kwargs)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask/views.py", line 107, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_restx/resource.py", line 46, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_restx/marshalling.py", line 244, in wrapper
    resp = f(*args, **kwargs)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_jwt_extended/view_decorators.py", line 153, in decorator
    verify_jwt_in_request(optional, fresh, refresh, locations, verify_type)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_jwt_extended/view_decorators.py", line 89, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_jwt_extended/view_decorators.py", line 323, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_jwt_extended/utils.py", line 127, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_jwt_extended/jwt_manager.py", line 553, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/flask_jwt_extended/tokens.py", line 94, in _decode_jwt
    decoded_token = jwt.decode(
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/jwt/api_jwt.py", line 168, in decode
    decoded = self.decode_complete(
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/jwt/api_jwt.py", line 136, in decode_complete
    self._validate_claims(
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/jwt/api_jwt.py", line 199, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/Users/macbook/Desktop/Return/Python/Code/Scissor_RestX/venv/lib/python3.9/site-packages/jwt/api_jwt.py", line 237, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2023-05-21 17:53:42 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:53:42 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:28 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:28 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:29 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:29 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:29 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:29 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:30 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:30 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:31 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:31 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:31 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:31 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:32 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:32 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:33 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:33 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:33 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:33 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 17:54:34 INFO: 127.0.0.1 - GetURLS endpoint called
2023-05-21 17:54:34 DEBUG: 127.0.0.1 - Retrieved 13 URLs
2023-05-21 18:11:57 INFO: 127.0.0.1 - ViewDeleteURLbyID a URL by UUID called
2023-05-21 18:11:57 DEBUG: 127.0.0.1 - Retrieved <url fcf45090-c432-4396-9c41-9950b575d340> URL by UUID
2023-05-21 18:32:50 INFO: 127.0.0.1 - CreateURL is called
2023-05-21 18:32:50 DEBUG: 127.0.0.1 - User <user admin@gmail.com> logged in
2023-05-21 18:37:54 INFO: 127.0.0.1 - Login is called
2023-05-21 18:37:54 ERROR: 127.0.0.1 - Invalid credentials
NoneType: None
2023-05-21 18:38:41 INFO: 127.0.0.1 - Login is called
2023-05-21 18:38:41 DEBUG: 127.0.0.1 - User <user admin@gmail.com> logged in
